docs/content/scripts/deskcrew.md

```
@@ -0,0 +1,110 @@
+    to: https://github.com/nuxt/scripts/blob/main/packages/script/src/runtime/registry/deskcrew.ts
+    size: xs
+  - label: "<ScriptDeskCrew>"
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/scripts/blob/main/packages/script/src/runtime/components/ScriptDeskCrew.vue
+    size: xs
+---
+
+[DeskCrew](https://deskcrew.io/) is a support widget combining live chat, AI answers from your
+knowledge base, a help centre and a changelog.
+
+Use [`useScriptDeskCrew()`{lang="ts"}](#usescriptdeskcrew){lang="ts"} for direct SDK calls, or
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)